pyvmc_csp.py

```diff
@@ -0,0 +1,38 @@
+import json
+import requests
+
+def getSddcs (strProdURL, orgID, sessiontoken):
+    myHeader = {'csp-auth-token': sessiontoken}
+    myURL = f"{strProdURL}/vmc/api/orgs/{orgID}/sddcs"
+    response = requests.get(myURL, headers=myHeader)
+    jsonResponse = response.json()
+    return jsonResponse
```

Comment:
We have invokeVMCGET() in VMCImportExport.py in the SDDC Import/Export project. Can we refactor the functions  even further? That function is in a class object so it already has session ID in it. The function also handles access token expiration issues which are a problem during long running operations.

It also has functions like invokeCSPGET

Something like this
def getSddcs (strProdURL, orgID, sessiontoken):
  myURL = f"{strProdURL}/vmc/api/orgs/{orgID}/sddcs"
  return invokeVMCGET(myURL)

Reply:
The point was the function referenced eliminates more redundant code.  i.e. we might want to pull that function into this common library. But it can be a second refactoring after all of this is done because it is a lot of work considering how the two projects merge to share common libraries.

Reply:
Ah I understand.  Yeah, I'm happy to look at that function and pull it into the common library.  Anything we can do to reduce the amount of duplicate code between the projects.